packages/showcases/core/csf/components/structure/card.csf.js

```diff
@@ -42,4 +42,13 @@ export const argTypes = {
       type: 'text',
     },
   },
+  headingLevel: {
+    type: { name: 'number', required: false },
+    description: 'Heading level of the card title',
+    control: {
+      type: 'range',
+      min: 1,
+      max: 6,
+    },
+  },
```

Comment:
You have to set the headingLevel required for react / vue but not required on svelte